modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeTableRowSkin.java

```diff
@@ -398,11 +358,34 @@ private void updateDisclosureNodeAndGraphic() {
 
     private void updateTableViewSkin() {
         TreeTableView<T> tableView = getSkinnable().getTreeTableView();
+        if (tableView != null) {
+            registerChangeListener(tableView.treeColumnProperty(), e -> {
+                // Fix for RT-27782: Need to set isDirty to true, rather than the
+                // cheaper updateCells, as otherwise the text indentation will not
+                // be recalculated in TreeTableCellSkin.leftLabelPadding()
```

Comment:
Actually this comment is not correct anymore since my PR (https://github.com/openjdk/jfx/pull/568).
Instead, it should be `TreeTableCellSkin.calculateIndentation()`.

Reply:
well .. that would be a merge conflict, had you updated the code comment in your PR 😁 As noted in my comments to Ajit's review, the listener registration is simply moved (including the code comment .. belatedly :)

Not sure how to handle it from here - following the rules, we might need a follow-up issue to the issue fixed in your PR?

Reply:
My PR is already merged, so this is not a problem. :)
I dont know, but since this is only fixing a (also before) wrong comment it might be okay? :)

Reply:
FYI: now the listener registration - including the incorrect code comment (which is the same as in current master) - is back at the old location in the re-inserted setupTreeTableViewListeners. So still need input whether it's okay to correct the code comment here.

Reply:
I think, it is okay to fix the comment in this review itself. It is very minor to warrant a separate PR. Also, this anomaly was discovered in this review makes it a candidate to be fixed here.

Reply:
done - thanks to both of you :)